### PR TITLE
Revamp the entire site using so-simple-theme

### LINKED ISSUE #1: Bora ressuscitar isso aqui?
@cadicallegari @lborguetti @boechat107

### PR DESCRIPTION
**Pull request by ryukinix**
Staging area: https://neowaylabs.lerax.me/

It assumes the following dependencies in a debian environment:

```
sudo apt install make cmake ruby-dev g++ libz-dev  -y
# it necessary ruby-bundler 2.0, debian only haves 1.13 on stable branch
sudo gem install bundler 
bundle install --path vendor/bundle
```

After getting the environment setup properly, to test it locally try: `make serve`.

Features:
+ Pre-built list of authors containings: me, boechat, cad, luciano,
katcipis and natel
+ Pre written post of me, katcipis and boechat (borrowed from boechat
site)
+ A simple front-end search engine for posts
+ Add profile pictures for all authors defined
+ Correct boilerplate for neoway, as well the business logo mark
+ Makefile to test it locally easily
+ Filtering posts by categories and tags
+ A about page
+ Integration with disqus
+ Integration with share buttons: linkdin, reddit, twitter, facebook

Future features based on Wishful thinking:
+ A authors pages with a brief description + pictures
+ ~Try use this boilerplate to write some post~
+ HackerNews share button
+ ~A Dockerfile to make the site more portable to another environment beyond GitHub~

Closes #1

### CODE REVIEW

**Comment by ryukinix** [comment]
If you have docker installed, there is a better way to run it after cloning the repo into this branch.
At the root of the repository run this

```
docker build -t neowaylabs . 
docker run --rm --network=host -it neowaylabs
```

The site will be available at http://localhost:4000

File: _data/authors.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+lborguetti:
+  name: Luciano Antonio Borguetti Faustino
+  picture: /images/authors/lborguetti.jpg
+  links:
+      - title: GitHub
+        url: https://github.com/lborguetti
+        icon: fab fa-github-square
+
+tiago4orion:
+  name: Luciano Antonio Borguetti Faustino
```
Comment (tiago4orion) [inline _data/authors.yml]: 😄

Comment (tiago4orion) [reply _data/authors.yml]: I'm not in the team anymore 😢

Comment (ryukinix) [reply _data/authors.yml]: Sorry for this mistake, copy & paste effects. I know this, but you have a lot of software here. Do you mind if I keep your name as author here? There is a LOT of things that you wrote here...

**Comment by ryukinix** [comment]
Galera posso dar merge nessa bagaça aqui? @neoway-labs

**Comment by ryukinix** [comment]
MERGING!!!!

**Pull request merged**